backends/gdx-backend-lwjgl/pom.xml

```diff
@@ -12,6 +12,13 @@
   <packaging>jar</packaging>
   <name>libGDX LWJGL Backend</name>
 
+  <repositories>
+    <repository>
+      <id>athou-repo</id>
+      <url>https://github.com/Athou/maven-repository/raw/master/repo</url>
```

Comment:
We cannot use this fork, because it does not contain the commit of the original repository that removes the LICENSE file.